api/src/routing.js

```diff
@@ -467,6 +467,7 @@ app.postJson('/api/v1/places/:id', function(req, res) {
     .catch(err => serverUtils.error(err, req, res));
 });
 
+app.get('/api/v1/place', place.v1.getAll);
```

Comment:
I'm now thinking it might be more appropriate to use the API endpoint 
/api/v1/places. This would better reflect the fact that we're returning a collection of places and align with standard REST API design practices.

Reply:
Same for the `person` endpoint.
@jkuester What do you think?